### PR TITLE
standardize dataset pipelines

### PR DESCRIPTION
**Pull request by sabinem**
The PR moves the pipeline from the `scripts` to `pipelines`, where each pipeline has its own directory

- a sample query is added
- spatial unit mapping and the pipeline steps are standardized
- metadata is added for table and columns
- a dataset sample is generated consisting of 40 randomly selected rows.

### CODE REVIEW

File: pipelines/A4/A4.R [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  dplyr::rename(
+    "verbrauchsart" = rubrik,
+    "anzahl" = tj
+  ) %>% tidyr::pivot_wider(
+    names_from = c("verbrauchsart"),
+    values_from = anzahl,
+    names_prefix = "terajoule_"
+  ) %>%
+  janitor::clean_names()
+colnames(ds$export)
```
Comment (cmdoret) [inline pipelines/A4/A4.R:35]:
```suggestion
colnames(ds$postgres_export)
```
(or maybe the line can be removed)

File: pipelines/A4/queries.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+-- Wieviele Einträge enthält die Tabelle?
```
Comment (cmdoret) [inline pipelines/A4/queries.sql:1]: The question is asked to the whole database, isn't it?
If so, here we would not know what table you are talking about

**Review by cmdoret — APPROVED** [review]
A5 looks good, I only had minor comments for A4

File: pipelines/A4/queries.sql (hunk 2 above)
Comment (sabinem) [reply pipelines/A4/queries.sql]: I have done a better query now.

File: pipelines/A4/queries.sql [hunk 3]
```diff
@@ -1,5 +1,8 @@
--- Wieviele Einträge enthält die Tabelle?
-SELECT COUNT(*)
+-- Welches war der am meisten importierte Energietraeger 2020 in der Schweiz?
+SELECT energietraeger as meist_importierte_enrgietraeger_2020
 FROM energiebilanz_schweiz_in_tera_joule AS T
 JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid
-WHERE S.name_de='Schweiz' AND S.country=TRUE;
+WHERE S.name_de='Schweiz' AND S.country=TRUE
+AND jahr='2020'
+GROUP BY terajoule_import
```
Comment (cmdoret) [inline pipelines/A4/queries.sql:7]: Do we need to `GROUP BY` here? It is a numeric value

**Pull request merged**